src/server/service/oppfolgingsplanService.ts

```diff
@@ -150,6 +151,44 @@ export async function getOppfolgingsplanerAG(accessToken: string) {
   handleSchemaParsingError("Arbeidsgiver", "Oppfolgingsplan", response.error);
 }
 
+export async function getDineSykmeldteMedSykmeldinger(
+  accessToken: string
+): Promise<Sykmeldt[]> {
+  const response = array(sykmeldtSchema).safeParse(
+    await get(
+      `${serverEnv.SYKMELDINGER_ARBEIDSGIVER_HOST}/api/v2/dinesykmeldte`,
+      { accessToken }
+    )
+  );
+
+  if (response.success) {
+    return response.data;
+  }
+
+  handleSchemaParsingError(
+    "Arbeidsgiver",
+    "sykmeldtSchema-med-sykmeldinger",
```

Comment:
Denne burde hete det samme som schemaet (sykmeldt) evt renames?

Reply:
Mulig, tenkte det kunne være greit å skille på hvilket API som feiler? Det er samme DTO som  brukes fra backend, men to ulike APIer med litt ulik data som fylles ut (sykmeldinger eller ikke)